recipes/master.rb

```diff
@@ -84,6 +80,16 @@
   end
 end
 
+ruby_block 'delay salt-master service start' do
+  block do
+  end
+  notifies :start, 'service[salt-master]'
```

Comment:
@shortdudey123 it is in the favor of LWRP resource that I am working on, e.g. `external_auth` and it works slightly better by delaying the service start.